dashboard/app/mail_bug.txt

```diff
@@ -56,7 +57,7 @@ If the bug is already fixed, let syzbot know by replying with:
 
 {{- if or .ReproCLink .ReproSyzLink}}
 
-If you want syzbot to run reproducer on another tree, reply with:
+If you want syzbot to run reproducer for you, reply with:
```

Comment:
I wouldn't say "for you", it's unnecessary and feels a bit... I don't know... out of place. This all process is generally not personally for the developer, somebody may also issue it for somebody else.
```
If you want syzbot to re-run the reproducer, reply with:
```